src/transaction/log_applier.c

```diff
@@ -8196,10 +8215,8 @@ la_apply_log_file (const char *database_name, const char *log_path, const int ma
       return error;
     }
 
-  for (i = 0; i < LA_NUM_DELAY_HISTORY; i++)
-    {
-      delay_hist[i] = -1;
-    }
+  init_delay_history (delay_hist);
```

Comment:
피드백 시 저도 prefix를 잊었네요.

```suggestion
  la_init_delay_history (delay_hist);
```

Reply:
@hornetmj 반영하였습니다